atlas/style.css

```diff
@@ -0,0 +1,16 @@
+/*
+Theme Name: Atlas
+Theme URI: https://github.com/MaggieCabrera/community-themes/tree/trunk/atlas
+Author: ajlende
```

Comment:
I think both in here and the changelog this should be the WordPress team once we move this repo the WP namespace, but we can change that in a followup

Reply:
Fixed 0871b68